a11y_pygments/utils/utils.py

```diff
@@ -1,7 +1,7 @@
 import logging
 import os
 from pathlib import Path
-from typing import List
+from typing import List, Union
```

Comment:
Since we have Python>= 3.9 we should be using the default `list` type
```suggestion
from typing import Union
```